### PR TITLE
Default coap config

### PR DESCRIPTION
**Pull request by sbernard31**

### CODE REVIEW

File: leshan-server-cf/src/main/java/org/eclipse/leshan/server/californium/LeshanServerBuilder.java [hunk 1]
```diff
@@ -268,6 +268,16 @@ public LeshanServerBuilder setDtlsConfig(DtlsConnectorConfig.Builder config) {
         return this;
     }
 
+    /**
+     * The default Californium/CoAP {@link NetworkConfig} used by the builder.
+     */
+    public static NetworkConfig createDefaultNetworkConfig() {
+        NetworkConfig networkConfig = new NetworkConfig();
+        networkConfig.set(Keys.MID_TRACKER, "NULL");
+        networkConfig.set(Keys.MAX_MESSAGE_SIZE, 4096);
```
Comment (msangoi) [inline leshan-server-cf/src/main/java/org/eclipse/leshan/server/californium/LeshanServerBuilder.java]: Same remark here: MAX_RESOURCE_BODY_SIZE? default instead?

File: leshan-server-demo/src/main/java/org/eclipse/leshan/server/demo/LeshanServerDemo.java [hunk 2]
```diff
@@ -233,7 +233,17 @@ public static void createAndStartServer(int webPort, String localAddress, int lo
         builder.setEncoder(new DefaultLwM2mNodeEncoder());
         LwM2mNodeDecoder decoder = new DefaultLwM2mNodeDecoder();
         builder.setDecoder(decoder);
-        builder.setCoapConfig(NetworkConfig.getStandard());
+
+        // Create CoAP Config
+        NetworkConfig coapConfig;
```
Comment (msangoi) [inline leshan-server-demo/src/main/java/org/eclipse/leshan/server/demo/LeshanServerDemo.java]: not passed to the builder

File: leshan-server-cf/src/main/java/org/eclipse/leshan/server/californium/LeshanBootstrapServerBuilder.java [hunk 3]
```diff
@@ -146,6 +146,16 @@ public LeshanBootstrapServerBuilder setDtlsConfig(DtlsConnectorConfig.Builder dt
         return this;
     }
 
+    /**
+     * The default Californium/CoAP {@link NetworkConfig} used by the builder.
+     */
+    public static NetworkConfig createDefaultNetworkConfig() {
+        NetworkConfig networkConfig = new NetworkConfig();
+        networkConfig.set(Keys.MID_TRACKER, "NULL");
+        networkConfig.set(Keys.MAX_MESSAGE_SIZE, 4096);
```
Comment (msangoi) [inline leshan-server-cf/src/main/java/org/eclipse/leshan/server/californium/LeshanBootstrapServerBuilder.java]: Same as client and server config.

**Comment by sbernard31** [comment]
Integrated in master (commit 49f483ad4001b288cb973d0653fca0b283bd6f98, 54d7d8f6cd4d5c2ad20977b84f041a3f1c11e249, d14a7c30ed497ed2ee6d15005aaab1106907b5df)

**Pull request closed without merging**